examples/quickstart/jupyter-notebooks/druidapi/druidapi/datasource.py

```diff
@@ -32,28 +32,28 @@ class DatasourceClient:
 
     See https://druid.apache.org/docs/latest/operations/api-reference.html#datasources
     '''
-    
+
```

Comment:
I gotta find the trick in VS Code that drops trailing spaces automatically...

Reply:
I use vim and configured that in my vimrc 😅

Reply:
Vim? Now that's true old-school!